components/closed-caption.json

```diff
@@ -0,0 +1,38 @@
+{
+  "closedcaption": {
+    "comment": "",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=40865%3A50204",
+    "background-light": "@theme-common-control-closedcaption-background-light",
+    "background-dark": "@theme-common-control-closedcaption-background-dark",
```

Comment:
we don't specify background light and dark. Just a background colour and the correct colour will be selected based on the thme